application/app/models/Playlist.java

```diff
@@ -0,0 +1,45 @@
+package models;
+
+import java.util.Objects;
+
+public class Playlist implements Comparable<Playlist> {
+
+    private PlaylistDto wrapped;
+
+    private Playlist(PlaylistDto wrapped) {
```

Comment:
Remove Dto postfix in DTO naming? The postfix is defined in build.sbt

./build.sbt:18:    openapiJavaConfig := JavaConfig.default.copy(json = JavaJson.jackson).addMapping(CustomMapping.forName({ case s => s + "Dto" })),

Reply:
It does not look so bad with the Dto postfix!

Reply:
This class is to extend the PlaylistDto with methods like a custom `equals()` and the `Comparable` interface. If you have a better idea how to make a `PlaylistDto` sortable, I'm open for proposals :-)

Reply:
Got it. I was more in to remove the Dto postfix, but I would leave it as it is for now.